crowdstrike/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.12-alpine
+ENV CONNECTOR_TYPE=EXTERNAL_IMPORT
```

Comment:
It's a default configuration for the collector, but it looks like a c/c mistake (aka copypasta) from possibly an existing file from opencti (hence "connector" and not collector).

This sets a defaults that overrides any default set in code, and I don't think it's necessary at all (rather, it would cause surprises)